apps/backend/src/modules/users/user.repository.ts

```diff
@@ -41,8 +48,16 @@ class UserRepository implements Repository {
 		return user ? UserEntity.initialize(user) : null;
 	}
 
-	public async findAll(): Promise<UserEntity[]> {
-		const users = await this.userModel.query().execute();
+	public async findAll({
+		page,
+		pageSize,
+	}: PaginationParameters): Promise<UserEntity[]> {
+		const offset = --page * pageSize;
```

Comment:
it's generally better to move calculations to the service rather than keeping them in the repository